pkgs/development/compilers/circt/default.nix

```diff
@@ -62,8 +63,8 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     runHook preInstall
-    mkdir -p $out/bin
-    mv bin/{{fir,hls}tool,circt-{as,dis,lsp-server,opt,reduce,translate}} $out/bin
+    mkdir -p $out
+    CMAKE_INSTALL_PREFIX=$out cmake --build . --target install --config Release
     runHook postInstall
   '';
```

Comment:
Can we just drop installPhase and let cmake do the install automatically?

Reply:
Thanks for the review!
I just tried removing the installPhase, and it seems to work.